fli-iam/shanoir-ng
# shanoir-issue#2372: add endpoint to remove SR from viewer

How to test:
Open DICOM in ohif viewer
Make a measurement
Open right-side panel
Click on "Create report":
![image](https://github.com/user-attachments/assets/a1a9f13c-02f0-4f50-891d-bdf54d7cc94f)

At this point, a new dataset is visible in solr:
![image](https://github.com/user-attachments/assets/b6ab9f05-d1a8-4a47-9c3b-7b494e1f952b)


In the left-side panel, a new serie appears:
![image](https://github.com/user-attachments/assets/c12ce837-b9f3-4d8e-a5ca-f7a4fba87f56)

Click on the trash-bin icon, a pop-up appears, click 'yes' to delete the measurement:
![image](https://github.com/user-attachments/assets/09acf692-aac8-4c6c-83ee-da18aa548585)

The solr dataset is deleted.
